cmd/mpd-mpris/main.go

```diff
@@ -84,8 +84,10 @@ func main() {
 			addr = "localhost"
 			detectLocalSocket()
 		} else {
-			if strings.Index(env_host, "@") > -1 {
-				addr_pwd := strings.Split(env_host, "@")
+			// When looking for the password delimiter, ignore the first character.
+			// An '@' sign at the start of the envvar signifies an "abstract socket" without password.
```

Comment:
Can you leave a link to the documentation of the `MPD_HOST` variable somewhere here? Would be great for someone else (me included) when they need to visit this part of the code again.

Reply:
Hmm... there are a few options. There's the libmpdclient sources themselves explicitly points out this case:
https://github.com/MusicPlayerDaemon/libmpdclient/blob/master/src/settings.c#L67

However a better source might be the MPC man page:
https://www.musicpd.org/doc/mpc/html/#envvar-MPD_HOST

which points you to the `--host` commandline option (internally they're parsed the same way).
https://www.musicpd.org/doc/mpc/html/#cmdoption-host

Reply:
I think the last one works ^^